docs/how-to/tcp-udp-ingress.md

```diff
@@ -2,6 +2,8 @@
 
 In a Kubernetes environment, an Ingress is an object that allows access to the Kubernetes services from outside the Kubernetes cluster. Standard Kubernetes Ingress resources assume that all the traffic is HTTP-based and does not cater to non-HTTP based protocols such as, TCP, TCP-SSL, and UDP. Hence, critical applications based on L7 protocols such as DNS, FTP, LDAP, and so on, cannot be exposed using standard Kubernetes Ingress.
 
+Please note that in addition to Ingress option, there is an alternate way to expose end user traffic to TCP applications, which is to use Kubernetes Service Type `LoadBalancer`. Please read more about how to use [Service Type LoadBalancer in Citrix ADC](https://github.com/citrix/citrix-k8s-ingress-controller/blob/master/docs/network/type_loadbalancer.md)
```

Comment:
The standard Kubernetes solution is to create a `service` of [`type LoadBalancer`](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#external-load-balancer-providers). You can find out more about [Service Type LoadBalancer in Citrix ADC](https://github.com/citrix/citrix-k8s-ingress-controller/blob/master/docs/network/type_loadbalancer.md). A second option is to annotate the `Ingress` object as described below.